tokens/index.js

```diff
@@ -24,8 +24,19 @@ export const font = {
   }
 };
 
+export const space = {
+  half: '4px',
```

Comment:
I'd prefer that too. The dot notation is why I ended up with half, or it would have just been .5.

The issue with shifting the whole scale is that 8 really is the base unit. 8, 16, 24, 32, 48, 64. x1, x2, x3, x4, x6, x8. The half is an exception for extra tight spaces because the next number down would be 0. Pretending like 4 is our base unit would make the scale x1, x2, x4, x8, x10, x12, x16. One of the main reasons for a scale here is language, to have the designers thinking in 8s and being able to communicate this to developers. 

I'm open to another suggestion but I couldn't think of any that were more clear than half.

Reply:
After some discussion we have agreed to stick with `half` for now. We can always revisit the name later.